firmware/Core/Src/transforms/arrays.c

```diff
@@ -63,6 +63,51 @@ void GEN_uint64_to_str(uint64_t value, char *buffer) {
     buffer[index] = '\0';
 }
 
+/// @brief Converts a int64_t to a string.
+/// @param value The input value to convert.
+/// @param buffer The output buffer to write the string to. MUST BE 32 BYTES LONG.
+void GEN_int64_to_str(int64_t value, char *buffer) {
+    // Ensure the buffer is valid and large enough
+    if (buffer == NULL) {
+        return;
+    }
+
+    char temp[32];  // Temporary array to hold digits
```

Comment:
You can remove the comment if you name the variable appropriately.

Reply:
Sure thing